projects/fusion-ui/components/table/common/entities/table-row-changed-data.ts

```diff
@@ -1,5 +1,6 @@
 export interface TableRowChangedData {
     rowIndex: string | number;
+    rowSpanIndex?: number;
```

Comment:
In the emitted event for row data changed we added it to know what element (index) in the data array was changed.